systests/rs-security/src/test/java/org/apache/cxf/systest/jaxrs/security/jose/jwejws/JAXRSJweJwsTest.java

```diff
@@ -85,8 +82,7 @@ public void testJweJwkPlainTextRSA() throws Exception {
     @Test
     public void testJweJwkBookBeanRSA() throws Exception {
         String address = "https://localhost:" + PORT + "/jwejwkrsa";
-        BookStore bs = createJweBookStore(address,
-                                       Collections.singletonList(new JacksonJsonProvider()));
+        BookStore bs = createJweBookStore(address, List.of());
```

Comment:
It turns out, Jackson for never used in serde process :)) With 2.18.x, it kicks in but conflicts with CXF's JSONProvider